python/lsst/analysis/tools/interfaces/datastore/_sasquatchDatastore.py

```diff
@@ -86,6 +86,16 @@ class SasquatchDatastore(GenericBaseDatastore):
     environment variable and "lsst.dm" if no environment variable is defined.
     """
 
+    extra_fields: dict[str, str | int | float] | None
+    """Extra key/value pairs that should be passed along with the metric
+    when storing in Sasquatch.
+
+    If not given extra key/value pairs can be obtained from the
+    ``$SASQUATCH_EXTRAS`` environment variable. This environment variable
+    must have the form of "k1=v2;k2=v2". If no environment variable is given
+    defaults can be read from the datastore configuration.
```

Comment:
Same comment about understanding what trumps what. Maybe:
```suggestion
    Overridden by the ``$SASQUATCH_EXTRAS`` environment variable if it is
    defined; this environment variable must have the form of "k1=v2;k2=v2".
    The key-value pairs in ``$SASQUATCH_EXTRAS`` and the datastore
    configuration are *NOT* merged.
```
(Though given my comment below, maybe merging was the intended behavior?)